addons/web_editor/views/snippets.xml

```diff
@@ -395,11 +395,14 @@
         <we-input class="o_we_large_input" string="Tooltip" data-set-title="" placeholder="Title tag"
             title="'Title tag' is shown as a tooltip when you hover the picture."/>
 
-        <we-row string="Transform">
+        <we-row string="Transform" class="o_we_full_row">
             <we-button data-crop="" data-no-preview="true" title="Crop Image"><i class="fa fa-crop"></i></we-button>
             <we-button data-transform="" data-no-preview="true" title="Transform the picture (click twice to reset transformation)">
                 <i class="fa fa-object-ungroup"></i>
             </we-button>
+            <we-button data-reset="" data-no-preview="true" title="Reset transformation" class="ml-auto">
+                Reset
+            </we-button>
```

Comment:
Not convinced by the spec here either. That button looks like it would reset crop and transform, not only transform don't you think?

Reply:
I agree, maybe it should reset both transforms, I am checking with SBU